packages/react-components/react-card/src/components/Card/useCard.ts

```diff
@@ -13,18 +13,28 @@ import { useFocusableGroup } from '@fluentui/react-tabster';
  * @param ref - reference to root HTMLElement of Card
  */
 export const useCard_unstable = (props: CardProps, ref: React.Ref<HTMLElement>): CardState => {
+  const { appearance = 'filled', focusMode = 'off' } = props;
+
+  const focusMap: Record<CardCommons['focusMode'], UseFocusableGroupOptions['tabBehavior']> = {
```

Comment:
using `Record` for exact dictionaries leads usually to problems as it creates N:N relationship unions which is incorrect.
![image](https://user-images.githubusercontent.com/1223799/166507415-14a86b78-04aa-4087-bcc8-cafd44f4ea99.png)

 Based on this shape following is correct:

![image](https://user-images.githubusercontent.com/1223799/166506558-512c4daa-b083-4883-b84c-8390bb4af1cb.png)

but in reality it is incorrect.

**What we should do:**

Try to not "overtype" our code and rely on actual API's that we use by leveraging `as const` assertion

This will provide exact shape which produces sound check when used within 3rd party API call. Although it wont provide you "nice intellisense" when writing this API, it is programatically 100% correct.

```ts
const focusMap = {
    off: 'limitedTrapFocus',
    'no-tab': 'limitedTrapFocus',
    'tab-exit': 'limited',
    'tab-only': 'unlimited',
  } as const;
```

Reply:
That's a good point, I'll make the change. I had initially gone with that approach but moved for type safety.

Reply:
well unsound "type-safety" is not type safety at all :D. ty